services/api/src/routes/game.ts

```diff
@@ -7,8 +8,14 @@ router.use((_req, res, next) => {
   next();
 });
 
-router.get('/test', async (_req, res) => {
-  res.json({ test: true });
+router.get('/:id', async (req, res) => {
+  const pathUUID = req.params.id;
+  const uuidValidationSchema = Joi.string().guid({
+    version: 'uuidv4',
+    separator: '-',
+  });
+  const authResult = uuidValidationSchema.validate(pathUUID);
+  !authResult.error ? res.json({ test: true }) : res.json({ test: false });
```

Comment:
I think the successful validation result could for now simply be `res.status(200).send({});` and in case of error `res.status(400).send{...validation error}`.